portfolio/src/main/webapp/script.js

```diff
@@ -32,3 +32,27 @@ function toggleText(divID) {
     var content = document.getElementById(divID);
     content.classList.toggle('visible');
 }
+
+function getData() {
+  fetch('/data').then(response => response.json()).then((fruit) => {
+    console.log(fruit.fruit0);
+    console.log(fruit.fruit1);
+    console.log(fruit.fruit2);
+
+    const fruitListElement = document.getElementById('data-container');
+    fruitListElement.innerHTML = 'Fruits: ' + fruit.fruit0 + ', ' + fruit.fruit1 + ', ' + fruit.fruit2;
```

Comment:
Once you move to arrays, it is completely fine to iterate over the array and manually concatenate, but `Array` structure has a `.join(separator)` method that may be helpful to get to a one-liner.